server/athenian/api/controllers/features/github/unfresh_pull_request_metrics.py

```diff
@@ -50,21 +51,21 @@ async def fetch_pull_request_facts_unfresh(done_facts: Dict[str, PullRequestFact
     :return: Map from PR node IDs to their facts.
     """
     add_pdb_hits(pdb, "fresh", 1)
-    blacklist = PullRequest.node_id.notin_any_values(done_facts)
+    blacklist = done_facts
```

Comment:
Wow-wow this affects prod much and not properly covered with tests, we cannot edit it.